x/mint/go.mod

```diff
@@ -0,0 +1,138 @@
+module cosmossdk.io/x/mint
+
+go 1.21.0
+
+require (
+	cosmossdk.io/api v0.7.1
+	cosmossdk.io/collections v0.4.0
+	cosmossdk.io/core v0.12.0
+	cosmossdk.io/depinject v1.0.0-alpha.4
+	cosmossdk.io/errors v1.0.0
+	cosmossdk.io/log v1.2.1
+	cosmossdk.io/math v1.1.2
+	cosmossdk.io/store v1.0.0-rc.0
+	github.com/cosmos/cosmos-proto v1.0.0-beta.3
+	github.com/cosmos/cosmos-sdk v0.47.5
```

Comment:
Mint should import latest SDK (pseudo version from main + replace)

Reply:
It does now.